src/plugins/home/public/application/components/home_list_card.tsx

```diff
@@ -41,10 +41,18 @@ export const LEARN_OPENSEARCH_CONFIG = {
     {
       label: 'Creating dashboards',
       href: 'https://opensearch.org/docs/latest/dashboards/dashboard/index/',
-      description: 'Build interactive dashboards to explore and analyze your data',
+      description: 'Build interactive dashboards to explore and analyze your data.',
     },
   ],
-  allLink: 'https://opensearch.org/docs/latest/',
+  allLink: (
+    <EuiLink href="https://opensearch.org/docs/latest/" target="_blank">
```

Comment:
should we b using the docLinks service? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/public/doc_links/doc_links_service.ts#L70

Reply:
Yes, good point. we should use docLinks if the link is availible.